virttest/libvirt_xml/vm_xml.py

```diff
@@ -2013,6 +2019,19 @@ def get_feature_list(self):
             feature_list.append(feature_node)
         return feature_list
 
+    def get_feature_index(self, name):
+        """
+        Get the feature's index in the feature list by given name
+
+        :param name: str, the feature's name
+        :return: int, the index of this feature in the feature list
+        """
+        feature_list = self.get_feature_list()
+        for i in range(0, len(feature_list)):
+            if self.get_feature(i).get('name') == name:
+                return i
```

Comment:
```suggestion
        return [ftr[name] for ftr in self.get_feature_list()].index(name)
```

Reply:
@Yingshun , thanks. But above codes has a minor issue with below error because 'ftr' is a Element feature type. 
`2021-07-08 23:35:32,397 test             L0915 ERROR| TypeError: list indices must be integers or slices, not str`

Finally I updated it to ` return [ftr.get('name') for ftr in self.get_feature_list()].index(name)`